models/answer.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(sequelize, DataTypes) {
+  let Answer = sequelize.define("Answer", {
+    member_id: {
```

Comment:
1) member_id should be: teammember_id  since the table name is TeamMember
2) Add the association between the two tables